ci/scripts/js_build.sh

```diff
@@ -30,7 +30,14 @@ yarn lint:ci
 yarn build
 
 if [ "${with_docs}" == "true" ]; then
-  yarn doc
+  if [ $(git config --get remote.origin.url) == "https://github.com/apache/arrow.git" ]; then
+    yarn doc
+  elif [ $(git config --get remote.upstream.url) == "https://github.com/apache/arrow.git" ]; then
+    yarn doc --gitRemote upstream
```

Comment:
Could you also add `apache` case?

```shell
  elif [ $(git config --get remote.apache.url) == "https://github.com/apache/arrow.git" ]; then
    yarn doc --gitRemote apache
```

We're using `apache` in our release document: https://cwiki.apache.org/confluence/display/ARROW/Release+Management+Guide

Reply:
Will do. It's also described at https://arrow.apache.org/docs/developers/contributing.html.

Reply:
Done in 192f828